packages/gitbook-docs/src/__tests__/parseContent.test.ts

```diff
@@ -164,4 +164,16 @@ describe('parseContent', () => {
       ),
     ]);
   });
+
+  xit('should parse quote', () => {
+    const quote =
+      '> _« ... al fine di consentire le attività di riconciliazione del pagamento... »_';
+    expect(parseContent(quote, config)).toStrictEqual([
+      new Markdoc.Tag('Quote', {}, [
+        new Markdoc.Tag('Paragraph', {}, [
+          '« ... al fine di consentire le attività di riconciliazione del pagamento... »',
```

Comment:
I see that the `_` char is missing, is the correct behaviour?

Reply:
Well, the parse wraps the text (without the `_`) in an `em` node, but I think we should both handle `> _« ... al fine di consentire le attività di riconciliazione del pagamento... »_` and `> « ... al fine di consentire le attività di riconciliazione del pagamento... »`